examples/ping-ring/main.go

```diff
@@ -95,12 +95,19 @@ func main() {
 
 	logger := log.StandardLogger()
 
+	rp, err := ringpop.New("ping-app",
+		ringpop.Channel(ch),
```

Comment:
Agreed that we should shoot for brevity, but not where it comprises on readability/understandability. I think we're tip-toeing that line here. This pattern reminds me slightly of Scala's Options (for example, http://danielwestheide.com/blog/2012/12/19/the-neophytes-guide-to-scala-part-5-the-option-type.html) and there it's clear that you are dealing with Options.

It's not so clear what the thing is that is living in the ringpop namespace if it's just called `ringpop.Channel()`. I think the name`ringpop.options` may be too literal and that `ringpop.config.Channel()` seems the _right_ level of abstraction.